rust/noosphere-sphere/src/petname/write.rs

```diff
@@ -6,9 +6,15 @@ use ucan::{crypto::KeyMaterial, store::UcanJwtStore, Ucan};
 
 use crate::{internal::SphereContextInternal, HasMutableSphereContext, SpherePetnameRead};
 
-fn validate_petname(petname: &str) -> Result<()> {
+fn validate_petname(
```

Comment:
Can we simplify this to disallowing any `did:` petname? That is probably always a bad idea because the user would be creating a misleading alias. And, it would save us from reaching into the `SphereContext` just to get a value to validate against.